codespell_lib/data/dictionary.txt

```diff
@@ -20874,6 +20893,7 @@ programable->programmable
 programatic->programmatic
 programatically->programmatically
 programattically->programmatically
+programed->programmed
```

Comment:
You could also add `programmd->programmed`.

Also not yours but I've just seen `programm->program` should also suggest the BE `programme`

Reply:
I wanted to add programme->program to enGB_to_enUS, 
but it is not in aspell http://app.aspell.net/lookup?dict=en_US&words=programm%0D%0Aprogramme

Reply:
It is if you look in the right dictionary? http://app.aspell.net/lookup?dict=en_GB-ise&words=program%0D%0Aprogramme

The way that I've always understood it, is that in the UK we have television programmes and computer programs, but in the US they use 'program' for both.

Reply:
Ok, I relied on the feature "show it to me, if it is in other dictionaries." which doesn't seem to work here. My local aspell test fails too.
let's try, what jenkins says to #1688